### DISCUSSION TITLE
generate_traffic.py vehicles spawned but not moving

### DISCUSSION THREAD

**Discussion by QFrankQ** [carla-simulator/carla#6694] [Q&A]
Hi, I was trying to get some cars moving around in the simulator. When I run generate_traffic.py, pedestrians and vehicles are spawned, but only the pedestrians are moving around and all vehicles are stationary. Has anyone encountered this problem before and how do you fix it?

**Comment by Liuzy0908** [comment]
more code details please.

**Reply by QFrankQ** [reply]
![Screenshot from 2023-08-07 16-26-36](https://github.com/carla-simulator/carla/assets/55074430/6de278ce-1f7c-48ee-a6b2-833f64748946)
![Screenshot from 2023-08-07 16-34-20](https://github.com/carla-simulator/carla/assets/55074430/38a42703-81c6-487a-953d-bd512217994e)
![Screenshot from 2023-08-07 16-34-42](https://github.com/carla-simulator/carla/assets/55074430/bf062131-aa74-42ca-bf97-c5cd0d0b950a)

Hi, here's the segment of code of generate_traffic.py that came with the package. https://github.com/carla-simulator/carla/blob/master/PythonAPI/examples/generate_traffic.py
I'm not super familiar about Actors and Blueprints. It seems like the vehicles are set to autopilot upon spawning, but unlike pedestrians the vehicles won't move.
